transformers/guild.ts

```diff
@@ -65,6 +65,12 @@ export function transformGuild(bot: Bot, payload: { guild: DiscordGuild } & { sh
         return [result.id, result];
       }),
     ),
+    members: new Collection(
+      payload.guild.members?.map((member) => {
```

Comment:
Can you check if voice states come with guild members and then also run the transformer for them

Reply:
Wouldn't that be member transformer's job?